caterva2/api.py

```diff
@@ -1092,10 +1152,19 @@ def remove(self):
         Examples
         --------
         >>> import caterva2 as cat2
-        >>> root = cat2.Root('@public')
-        >>> file = root['ds-1d.b2nd']
+        >>> import numpy as np
+        >>> # To remove a file you need to be a registered user
+        >>> urlbase = 'https://cloud.caterva2.net/demo'
+        >>> root = cat2.Root('@personal', urlbase, dict(username='user@example.com', password='foo'))
+        >>> path = 'root-example/dir2/ds-4d.b2nd'
+        >>> root.upload(path)
+        <Dataset: @personal/root-example/dir2/ds-4d.b2nd>
+        >>> auth_cookie = cat2.get_auth_cookie(urlbase, dict(username='user@example.com', password='foo'))
+        >>> file = cat2.File(path, '@personal', urlbase, auth_cookie)
         >>> file.remove()
-        '@public/ds-1d.b2nd'
+        '@personal/root-example/dir2/ds-4d.b2nd'
```

Comment:
For what is worth, here it is an example of using a complete path for identifying a file.  I think this is the way to go.  I'll open a ticket and hopefully we will put some consistency in reported paths through the API eventually.

Reply:
En the end, only the `api/list` has the behavior of not listing the complete path, but this is on purpose, and only when a parameter is passed.  So, do not pay attention to the above comment.